### PR TITLE
Put the five executables in a list

### PR DESCRIPTION
**Pull request by cclauss**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: docs/Homebrew-and-Python.md [hunk 1]
```diff
@@ -9,7 +9,12 @@ Homebrew provides formulae to brew a more up-to-date Python 2.7.x and 3.x.
 **Important:** If you choose to install a Python which isn't either of these two (system Python or brewed Python), the Homebrew team can only provide limited support.
 
 ## Python 2.x or Python 3.x
-Homebrew provides a formula for Python 2.7.x and one for Python 3.x. They don't conflict, so they can both be installed. The executable `python2` will always point to the 2.x and `python3` to the 3.x version. The executable `python` will always point to the macOS system Python.
+Homebrew provides one formula for Python 2.7.x and another for Python 3.x. The executables are organized as follows so that Python 2 and Python 3 can both be installed without conflict:
+* `python` will always point to the macOS system Python
+* `python2` will point to the brew-installed Python 2.7.x
+* `python3` will point to the brew-installed Python 3.x
+* `pip2` will point to the pip associated with the brew-installed Python 2.7x
```
Comment (apjanke) [inline docs/Homebrew-and-Python.md]: "2.7.x" instead of "2.7x".

**Comment by apjanke** [comment]
Changes look good now.

Do you mind squashing this PR to a single commit to keep our commit history tidy?

File: docs/Homebrew-and-Python.md [hunk 2]
```diff
@@ -9,7 +9,12 @@ Homebrew provides formulae to brew a more up-to-date Python 2.7.x and 3.x.
 **Important:** If you choose to install a Python which isn't either of these two (system Python or brewed Python), the Homebrew team can only provide limited support.
 
 ## Python 2.x or Python 3.x
-Homebrew provides a formula for Python 2.7.x and one for Python 3.x. They don't conflict, so they can both be installed. The executable `python2` will always point to the 2.x and `python3` to the 3.x version. The executable `python` will always point to the macOS system Python.
+Homebrew provides one formula for Python 2.7.x and another for Python 3.x. The executables are organized as follows so that Python 2 and Python 3 can both be installed without conflict:
+* `python` will always point to the macOS system Python
```
Comment (DomT4) [inline docs/Homebrew-and-Python.md]: I appreciate the wording exists already but "always" seems misleading here given:
```
This formula installs a python2 executable to /usr/local/bin.
If you wish to have this formula's python executable in your PATH then add
the following to ~/.zshrc:
  export PATH="/usr/local/opt/python/libexec/bin:$PATH"
```
Perhaps, "by default" instead.

**Comment by ilovezfs** [comment]
I think the existing wording is more succinct and equally clear, so I'd prefer we not make this change, as it only makes the text more repetitive.

**Comment by cclauss** [comment]
Closing in favor of #2963

**Pull request closed without merging**